scripts/css/fonts.css

```
@@ -0,0 +1,20 @@
+@font-face {
+  font-family: "SageUI";
+  font-style: normal;
+  font-weight: 400;
+  src: local("SageUI Regular") local("SageUI-Regular") url("../assets/SageUI-Regular.tff") format("truetyp");
```

```suggestion
  src: local("SageUI Regular") local("SageUI-Regular") url("../assets/SageUI-Regular.tff") format("truetype");
```

There's an "e" missing in the end